### PR TITLE
feat: created login page

### LINKED ISSUE #14: [FEAT] :sparkles: Create UI for `login` page
### Description

image

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Tchakoumi**
### Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Checklist:

- [ ] This pull request follow our contribution guidelines
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (postman and swagger)
- [ ] I have written tests for the code
- [ ] I have updated .env.example file for new .env variables

### What is current behavior?

Closes #(issue_number)

### CODE REVIEW

**Comment by Tchakoumi** [comment]
image

File: src/pages/login/Login.tsx [hunk 1]
```diff
@@ -1,7 +1,140 @@
...
+			} catch (err) {
+				// TODO: show feedback on UI
+			}
+		},
+	});
+
+	/**
+	 * Weather password should be visible or not
+	 * @constant
+	 * @author KanLSK
```
Comment (aayushchugh) [inline src/pages/login/Login.tsx]: please add yourself as `author`

**Review by aayushchugh — CHANGES_REQUESTED** [review]

File: src/pages/login/Login.tsx (hunk 1 above)
Comment (Tchakoumi) [reply src/pages/login/Login.tsx]: I've done it

**Review by aayushchugh — APPROVED** [review]

File: src/pages/login/Login.tsx [hunk 2]
```diff
@@ -1,7 +1,156 @@
...
+					<div className="hidden w-fit items-center gap-1 md:flex">
+						<img src={logo} alt="logo" className="w-[40px]" />
+						<p>Multi Email</p>
+					</div>
+					{/*form*/}
+					<div className="flex w-[90%] flex-col gap-2 text-[#3F72AF]">
+						<label htmlFor="email" className="text-[14px]">
+							Email
+						</label>
+						<input
```
Comment (Karol-Ari) [inline src/pages/login/Login.tsx:106]: Check the font and its weight

File: src/pages/login/Login.tsx [hunk 3]
```diff
@@ -1,7 +1,156 @@
...
+							type="email"
+							id="email"
+							{...formik.getFieldProps("email")}
+							className="rounded-md px-2 py-1 text-[12px] placeholder-[#3f72af]"
+							placeholder="example@multiemail.us"
+						/>
+					</div>
+					<div className="relative flex w-[90%] flex-col gap-2 text-[#3F72AF]">
```
Comment (Karol-Ari) [inline src/pages/login/Login.tsx:114]: Check the font and its weight

File: src/pages/login/Login.tsx (hunk 2 above)
Comment (Karol-Ari) [inline src/pages/login/Login.tsx:106]: Check the font and its weight

File: src/pages/login/Login.tsx (hunk 3 above)
Comment (Karol-Ari) [inline src/pages/login/Login.tsx:114]: Check the font and its weight

File: src/pages/login/Login.tsx [hunk 4]
```diff
@@ -1,7 +1,156 @@
...
+								onClick={() => setIsPasswordVisible(false)}
+							/>
+						) : (
+							<AiFillEye
+								className="absolute right-2 top-[60%]"
+								onClick={() => setIsPasswordVisible(true)}
+							/>
+						)}
+					</div>
+					<button
```
Comment (Karol-Ari) [inline src/pages/login/Login.tsx:137]: Check the font and its weight

File: src/pages/login/Login.tsx [hunk 5]
```diff
@@ -1,7 +1,156 @@
...
+						type="submit"
+						className="rounded-md bg-[#5271ff] px-2 py-1 text-[14px] text-white"
+					>
+						Log In
+					</button>
+					<p className="text-[12px]">
```
Comment (Karol-Ari) [inline src/pages/login/Login.tsx:143]: Check the font and its weight

**Review by Karol-Ari — CHANGES_REQUESTED** [review]
Overall good work, but check the font

File: src/pages/login/Login.tsx (hunk 2 above)
Comment (aayushchugh) [reply src/pages/login/Login.tsx:106]: @Karol-Ari what should be the font size and weight?

**Pull request merged**